flask_multipass_cern.py

```diff
@@ -351,6 +346,7 @@ def search_identities_ex(self, criteria, exact=False, limit=None):
         except RequestException:
             self.logger.warning('Refreshing identities failed for criteria %s (could not get API token)', criteria)
             if use_cache and cached_data:
+                self.cache.delay_refresh(cache_key, CACHE_TTL)
                 return cached_results, cached_data[1]
             else:
                 self.logger.error('Getting identities failed for criteria %s (could not get API token)', criteria)
```

Comment:
This block is not covered by the tests. I'm not sure how likely it is that this will raise but maybe we should add a test for it?

Reply:
added a test for it